### DISCUSSION TITLE
OpenGl support

### DISCUSSION THREAD

**Discussion by bekraoui** [bevyengine/bevy#3037] [Q&A] [answered]
Hi;
Is there any way to use bevy 0.5 with opengl (i know it will be supported by 0.6) i have tested the bevy pipelined branch, it does not support it yet?

I want also to thank you for the great effort you put in making this beautiful tool.

**Comment by kirawi** [comment, marked as answer]
Don't think wgpu supports opengl.

**Reply by bjorn3** [reply]
The latest wgpu supports opengl and webgl. This version is used by the new renderer on the piplined-renderer branch (bevy_render2, not bevy_render which still exists on this branch). The version of wgpu used by bevy 0.5 and by the main branch doesn't support opengl and webgl.
